src/common/sha3/xkcp_sha3x4.c

```diff
@@ -59,11 +62,16 @@ static void Keccak_X4_Dispatch(void *state) {
 	Keccak_X4_Permute_ptr = &KeccakP1600times4_PermuteAll_24rounds;
 	Keccak_X4_ExtractBytes_ptr = &KeccakP1600times4_ExtractBytes;
 #endif
-
-	(*Keccak_X4_Initialize_ptr)(state);
 }
 
 static void keccak_x4_inc_reset(uint64_t *s) {
+#if OQS_USE_PTHREADS_IN_LIBOQS
```

Comment:
Why is this guard of "dispatch_once_control" using a different #define than "CMAKE_USE_PTHREADS_INIT"?

Reply:
Because I made a mistake. Thanks for the catch!

Reply:
UW. Seems to have resolved the last CI problem. Please feel free to move this to Ready for Review when CI passes.

Reply:
I could mark Ready for Review, but I'm not sure if I should wait for #1576 to be merged first as a separate logical unit